Assignment2 (#1)
Abhi-at-Josh/Assignment: question1.txt

Q1. How to set multiple remote repositories for the same project? Explain the use case. you should have a working demo of it.
Ans:-

>     To set multiple remote repo for the same roject is helpful in verious scenerios 

Reviewer: ```suggestion
    To set multiple remote repo for the same project is helpful in various scenarios 
```
Typo

Author: I will check the grammar and spelling before submitting.

    such as

>     -push changes o diff platform provider(github,gitlab,bitbuket)

Reviewer: ```suggestion
    -push changes to different platform provider(github,gitlab,bitbuket)
```
Typo

Author: Change the typo and made it correct

      at same time
    -work in collaboration to diff team and organization at same time

>     -also to set up backup of the perticular data/repo on diff platform in case of any problems

Reviewer: ```suggestion
    - also to set up backup of a particular data/repo on different platform in case of any problems
```
Please avoid using these shortcuts like diff that we use while texting 😁 Make sure the first alphabet of the sentence starts will capital letter.

Author: I  will make sure to avoid using shortcuts or informal language in the future.

    git remote add origin https://github.com/yourusername/my-project.git

    add first remote 
      git remote add origin https://github.com/yourusername1/my-project.git
    add  second remote

>        git remote add origin https://github.com/yourusername2/my-project.git

Reviewer: >How to set multiple remote repositories for the same project?

I think the question is about repositories of the same project.
The above two looks like a different project. Is `git remote add origin https://github.com/yourusername1/my-project.git` and `https://github.com/yourusername2/my-project.git` repositories of the same project.

Author: Yes ,the repositories is of same project
I have made the change required
